pandas/core/arrays/base.py

```diff
@@ -610,3 +615,91 @@ def _ndarray_values(self):
         used for interacting with our indexers.
         """
         return np.array(self)
+
+
+def ExtensionOpsMixin(include_arith_ops, include_logic_ops):
```

Comment:
Is there a compelling reason to make a factory for this instead of just making two mixin classes?

```
class ExtensionArithmeticMixin(...):

class ExtensionLogicMixin(...):
```
?

Reply:
@jbrockmendel I am sharing the code of the operator between arithmetic and comparison operators.  Alternatively, I could just create a base mixin class, and have `ExtensionArithmeticMixin` and `ExtensionLogicMixin` inherit from the base.

If you guys want me to make that change, let me know.

Reply:
@jbrockmendel I implemented your suggestion.  It does look cleaner.  It's in the latest commit